### PR TITLE
fix: Update suggested replies logic

### PR DESCRIPTION
**Pull request by liamcho**
Fixes [AC-4063](https://sendbird.atlassian.net/browse/AC-4063)

### 요약
일반 봇이 메세지를 보내면 channel.lastMessage 가 업데이트 안된답니다. 그래서 channel.lastMessage 에 의존하는 코드를 수정했습니다.

### Changelogs
- improved stability

[AC-4063]: https://sendbird.atlassian.net/browse/AC-4063?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/modules/Channel/components/Message/index.tsx [hunk 1]
```diff
@@ -51,7 +52,7 @@ const Message = (props: MessageProps) => {
         && (
           config?.groupChannel?.showSuggestedRepliesFor === 'all_messages'
             ? true
-            : message.messageId === currentGroupChannel?.lastMessage?.messageId
+            : message.messageId === allMessages[allMessages.length - 1].messageId
```
Comment (HoonBaek) [inline src/modules/Channel/components/Message/index.tsx:55]: It will be good if we comment why we don't use `lastMessage` here

Comment (liamcho) [reply src/modules/Channel/components/Message/index.tsx:55]: showSuggestedRepliesFor can have two values: all_messages, and last_message_only. if last_message_only, we render only if message is the last one.

Comment (HoonBaek) [reply src/modules/Channel/components/Message/index.tsx:55]: I'm sorry, I was talking about leaving a comment on the line 😅

File: src/modules/GroupChannel/components/Message/index.tsx [hunk 2]
```diff
@@ -45,8 +45,9 @@ export const Message = (props: MessageProps): React.ReactElement => {
 
     const lastMessageInView = messages[messages.length - 1];
```
Comment (HoonBaek) [inline src/modules/GroupChannel/components/Message/index.tsx:46]: good to add here too

**Review by HoonBaek — APPROVED** [review]

**Pull request merged**